terraso_backend/apps/project_management/graphql/projects.py

```diff
@@ -55,30 +55,17 @@
 from apps.project_management.models.sites import Site
 from apps.soil_id.models.project_soil_settings import ProjectSoilSettings
 
-
-class UserRole(graphene.Enum):
-    viewer = collaboration_roles.ROLE_VIEWER
-    contributor = collaboration_roles.ROLE_CONTRIBUTOR
-    manager = collaboration_roles.ROLE_MANAGER
+ProjectRoleEnum = graphene.Enum(
+    "ProjectManagementProjectRoleChoices", [(c[0], c[1]) for c in ProjectRole.choices]
+)
```

Comment:
Can we have a comment explaining what this code is doing? Even within the confines of graphene APIs it seems pretty opaque.

Reply:
they have an API to do this! updated the code. and my opaque tomfoolery was causing a bug anyway so good catch